IPython/core/magics/execution.py

```diff
@@ -71,23 +72,27 @@ class TimeitResult(object):
 
     """
 
-    def __init__(self, loops, repeat, best, worst, all_runs, compile_time, precision):
+    def __init__(self, loops, repeat, average, stdev, all_runs, compile_time, precision):
         self.loops = loops
         self.repeat = repeat
-        self.best = best
-        self.worst = worst
+        self.average = average
+        self.stdev = stdev
```

Comment:
The "problem"? with that is that if the statistic functions are methods, then we will be calculating the standard deviation and the average **unless** we cached those at least. This is because we need those values to print its __str__ or __repr__ representation. Should I use this approach or the calculation of this two values should be done in the __init__ method?